samples/sp-webpart/src/webparts/mgtDemo/MgtDemoWebPart.ts

```diff
@@ -2,18 +2,22 @@ import * as React from 'react';
 import * as ReactDom from 'react-dom';
 import { Version } from '@microsoft/sp-core-library';
 import { BaseClientSideWebPart } from '@microsoft/sp-webpart-base';
-import {
-  IPropertyPaneConfiguration,
-  PropertyPaneTextField
-} from '@microsoft/sp-property-pane';
+import { IPropertyPaneConfiguration, PropertyPaneTextField } from '@microsoft/sp-property-pane';
+
+// import web component polyfills for browsers that need them
+import 'regenerator-runtime/runtime';
+import 'core-js/es/number';
+import 'core-js/es/math';
+import 'core-js/es/string';
+import 'core-js/es/date';
+import 'core-js/es/array';
+import 'core-js/es/regexp';
```

Comment:
In my testing these were not necessary - did you have a different experience?

Reply:
Yes... especially for our sample, such as the person-component still not loading: 

![issues](https://user-images.githubusercontent.com/11411686/85902328-a3e85800-b7b8-11ea-8882-16b86bb735c2.PNG)